### PR TITLE
Udpate sbo app stack odo blog

### PR DESCRIPTION
**Pull request by mezarin**
Following the documentation of creating a PR off of prod branch and merging it into the draft branch.

This is the commit to for this update: 
https://github.com/OpenLiberty/blogs/pull/1750/commits/7add1c3f9ee2c6cdac90f05b1760f6c79e30e3af

### CODE REVIEW

**Review by dmuelle — DISMISSED** [review]
LGTM- unrelated files are already in draft

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+[source,sh]
+----
+odo project create service-binding-demo
+----
+
+Use the OpenShift console (`Operators->OperatorHub`) to install this operator. Be sure to pick `service-binding-demo` as the namespace in which to install it.
+
+=== Provide service resource access to the Service Binding Operator
+
+Starting with link:https://github.com/redhat-developer/service-binding-operator/releases/tag/v0.10.0[v0.10.0], the Service Binding Operator, requires explicit permissions to access service resources.
```
Comment (scottkurz) [inline posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: Operator, requires => Operator requires

**Review by scottkurz — DISMISSED** [review]
Very minor grammar comment but approving

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc (hunk 1 above)
Comment (mezarin) [reply posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: Good catch. Updated. Thanks.

**Review by dmuelle — APPROVED** [review]

**Pull request merged**